pkgs/applications/graphics/tesseract/default.nix

```diff
@@ -1,67 +1,91 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig
+{ stdenv, fetchurl, fetchFromGitHub, autoreconfHook, pkgconfig
 , leptonica, libpng, libtiff, icu, pango, opencl-headers
 # Supported list of languages or `null' for all available languages
 , enableLanguages ? null
 # if you want just a specific list of languages, optionally specify a hash
 # to make tessdata a fixed output derivation.
 , enableLanguagesHash ? (if enableLanguages == null # all languages
-                         then "1h48xfzabhn0ldbx5ib67cp9607pr0zpblsy8z6fs4knn0zznfnw"
+                         then "11bi1hj2ihqrgvi9cam8mi70p4spm3syljkpnbglf4s8jkpfn15a"
```

Comment:
Is this default ever used? If `enableLanguages` is `null`, `languages.all` is passed through without any extra processing…

Reply:
Your're right, it's never used.
If we switch to single lang derivations only, we can get rid of `enableLanguagesHash ` completely. Otherwise, we should remove the unused default value.

Reply:
I think if selected-languages case works, killing it might be suboptimal: there are some substantially  two-language texts.

Reply:
I meant that if we switch to single lang derivations only, `tessdata` could be just a list of fixed-output lang derivations, so no extra output hash would be needed.